griptape/utils/stream.py

```diff
@@ -34,10 +34,13 @@ class Stream:
 
     @structure.validator  # pyright: ignore[reportAttributeAccessIssue]
     def validate_structure(self, _: Attribute, structure: Structure) -> None:
-        from griptape.config import config
+        from griptape.tasks import PromptTask
 
-        if not config.drivers.prompt.stream:
-            raise ValueError("prompt driver does not have streaming enabled, enable with stream=True")
+        streaming_tasks = [
+            task for task in structure.tasks if isinstance(task, PromptTask) and task.prompt_driver.stream
+        ]
+        if not streaming_tasks:
+            raise ValueError("Structure does not have any streaming tasks, enable with stream=True")
```

Comment:
looking at this again, what would happen if you had one streaming task and the others were not? would the utility still work?

Reply:
Yes it would still work, it would just "hang" while working on non-streaming tasks.